src/api/app/views/webui2/webui/repositories/_repository_card_content.html.haml

```diff
@@ -0,0 +1,49 @@
+- number_paths = 3
+.card-body
+  - if repository.path_elements.present?
+    Repository paths:
+    %ul.list-unstyled.ml-3
+      - repository.path_elements.each_with_index do |path, index|
+        - if index < number_paths
+          = render partial: 'repository_path_item', locals: { project: project, repository: repository, path: path }
+        - else
+          - content_for :repository_path_item do
+            = render partial: 'repository_path_item', locals: { project: project, repository: repository, path: path }
+      - if repository.path_elements.size > number_paths
+        %details.repository-path
+          %summary.small
+            %span.more More
+            %span.less Less
+          = yield :repository_path_item
+
+.card-footer.text-center
+  .row
+    - if User.current.can_modify?(project) && !repository.is_dod_repository?
+      .col
+        = link_to('#', title: 'Edit Repository',
+                       data: { toggle: 'modal', target: "#edit-repository-#{repository.id}" }) do
+          %i.fas.fa-edit.text-secondary{ alt: 'Edit Repository icon' }
+      .col
+        = link_to('#', title: 'Add Repository Path',
+                       data: { toggle: 'modal', target: "#add-repository-path-#{repository.id}" }) do
+          %i.fas.fa-plus-circle.text-primary{ alt: 'Add Repository Path icon' }
+    .col
+      - url = "#{download_url}/#{project.to_s.gsub(/:/, ':/')}/#{repository}"
+      = link_to(url, title: 'Download Repository') do
+        %i.fas.fa-download.text-secondary{ alt: 'Download Repository icon' }
+
+    - if User.current.can_modify?(project)
+      .col
+        = link_to('#', title: 'Delete Repository',
+                       data: { toggle: 'modal', target: "#delete-repository-#{repository.id}" }) do
+          %i.fas.fa-trash.text-danger{ alt: 'Delete Repository icon' }
```

Comment:
It should be using the same icon as the other delete actions, so `fa-times-circle` instead of `fa-trash`

Reply:
We have two different delete actions in this page and it would be nice to differentiate them. I would prefer to keep that icon

Reply:
We also have two different delete actions on the package show page and we still went with the same icon in both cases. This is because users expect a delete action when they see the `fa-times-circle` action (once they have seen it once). This is to be consistent

Reply:
I think the other icon is also really clear. User also expect the same icon to do the same thing, and here there are doing different things, deleting different things

Reply:
Anyway, it is change already. We can discuss this in another PR